app/controllers/contacts_controller.rb

```diff
@@ -0,0 +1,25 @@
+class ContactsController < ApplicationController
+
+  def new
+    @contact = Contact.new
+    @company = Company.find(params[:company_id])
+  end
+
+  def create
+    @company = Company.find(params[:company_id])
+    @contact = Contact.create(contact_params)
```

Comment:
is this contact associated with this company? if so, where are we assigning the `company_id` in the new `Contact` object?